mappings/net/minecraft/server/command/LootCommand.mapping

```diff
@@ -49,7 +49,7 @@ CLASS net/minecraft/class_3039 net/minecraft/server/command/LootCommand
 	METHOD method_13206 addTargetArguments (Lcom/mojang/brigadier/builder/ArgumentBuilder;Lnet/minecraft/class_3039$class_3042;)Lcom/mojang/brigadier/builder/ArgumentBuilder;
 		ARG 0 rootArgument
 		ARG 1 sourceConstructor
-	METHOD method_13207 getBlockInventory (Lnet/minecraft/class_2168;Lnet/minecraft/class_2338;)Lnet/minecraft/class_1263;
+	METHOD method_13207 getBlockItemContainer (Lnet/minecraft/class_2168;Lnet/minecraft/class_2338;)Lnet/minecraft/class_1263;
```

Comment:
Really `getBlockInventory` as this item container is within the block

Reply:
This PR uses "inventory" for entities, not blocks (like vanilla).